### ISSUE TITLE
Using with React (or Preact) + Typescript

### ISSUE THREAD

**Issue by makoven** [material-components/material-components-web-components#2253]
Hello. Is it possible to get typing for use inside JSX components?
Currently showing an error "Property 'mwc-button' does not exist on type 'JSX.IntrinsicElements'"

![Screenshot from 2021-03-15 17-02-44](https://user-images.githubusercontent.com/3522914/111122219-11d7e800-85b1-11eb-886a-f7b5a0f6e433.png)

**Comment by makoven** [comment]
Why not add declarations like these and write about compatibility with react?

https://github.com/wokwi/wokwi-elements/blob/ca2f4135e5f9bd3e021d21f633b12a2f6f2eb0e3/src/react-types.ts

This will instantly increase the popularity of the package)

**Comment by e111077** [comment]
Hello, what that package is doing is correct and should be done in user-end projects.

Though we would like to support all frameworks, directly supporting each individual framework may prove to be out of scope for our project. Additionally, we do intend to improve our documentation for react before we release this project as 1.0, but also we are waiting for this month's release of Lit2 which will have direct react support e.g. run a lit-element as a native react component.

TL;DR we are working on react integration DX and it will come before 1.0
